src/media/js/elements/nav.js

```diff
@@ -0,0 +1,149 @@
+define('elements/nav',
```

Comment:
@mstriemer I think it's okay to surrender that consumers of marketplace-elements are consumers of mcm?